BenchmarkVCFs/FindSamplesAndBenchmark.wdl

```diff
@@ -350,59 +362,60 @@ task MakeStringMap {
 }
 
 
-task FilterSymbolicAlleles { 
-  input {
-    File monitoring_script
-    File input_vcf
-    File input_vcf_index
-    String output_basename
-    Int disk_size
-    File ref_fasta
-    File ref_fasta_index
-    File ref_fasta_dict
-    Int preemptible_tries
-    Boolean no_address
-  }
-  command <<<
-    bash ~{monitoring_script} > monitoring.log &
-   
-    set -e
-
-    gatk --java-options "-Xmx10g" LeftAlignAndTrimVariants \
+task FilterSymbolicAlleles {
+    input {
+        File monitoring_script
+        File input_vcf
+        File input_vcf_index
+        String output_basename
+        Int disk_size
+        File ref_fasta
+        File ref_fasta_index
+        File ref_fasta_dict
+        Int preemptible_tries
+        Boolean no_address
+        String gatkTag
+    }
+    command <<<
+        bash ~{monitoring_script} > monitoring.log &
+
+        set -e
+
+        gatk --java-options "-Xmx10g" LeftAlignAndTrimVariants \
         -V  ~{input_vcf} \
         -O ~{output_basename}.tmp1.vcf.gz \
         --reference ~{ref_fasta} \
-        --split-multi-allelics  
-
-    rm ~{input_vcf}
-
-    gatk --java-options "-Xmx10g"  SelectVariants \
-          -V ~{output_basename}.tmp1.vcf.gz \
-          -O ~{output_basename}.tmp2.vcf.gz \
-          --remove-unused-alternates
-
-    rm ~{output_basename}.tmp1.vcf.gz
-
-    gatk --java-options "-Xmx10g" SelectVariants \
-          -V ~{output_basename}.tmp2.vcf.gz \
-          -O ~{output_basename}.vcf.gz \
-          --exclude-non-variants \
-          --select-type-to-exclude SYMBOLIC
-  >>>
-  runtime {
-    preemptible: preemptible_tries
-    memory: "12 GB"
-    cpu: "1"
-    disks: "local-disk " + ceil(disk_size) + " HDD"
-    docker: "broadinstitute/gatk:4.1.4.1" 
-    noAddress: no_address
-    maxRetries: 1
-  }
-  output {
-    File output_vcf = "~{output_basename}.vcf.gz"
-    File output_vcf_index = "~{output_basename}.vcf.gz.tbi"
-    File monitoring_log = "monitoring.log"
-  }
+        --split-multi-allelics
+
+        rm ~{input_vcf}
+
+        gatk --java-options "-Xmx10g"  SelectVariants \
+        -V ~{output_basename}.tmp1.vcf.gz \
+        -O ~{output_basename}.tmp2.vcf.gz \
+        --remove-unused-alternates
+
+        rm ~{output_basename}.tmp1.vcf.gz
+
+        gatk --java-options "-Xmx10g" SelectVariants \
+        -V ~{output_basename}.tmp2.vcf.gz \
+        -O ~{output_basename}.vcf.gz \
+        --exclude-non-variants \
+        --select-type-to-exclude SYMBOLIC
+    >>>
+    runtime {
+        preemptible: preemptible_tries
+        memory: "12 GB"
+        cpu: "1"
+        disks: "local-disk " + ceil(disk_size) + " HDD"
+        docker: "us.gcr.io/broad-gatk/gatk:"+gatkTag
```

Comment:
Just a suggestion, but in some places, when the docker string can change, it is either calculated like the above in the task -or- it's just a variable.  I know this idiom is carried over, but I think it would be great if there was a paragraph at the top that calculated and finalized all docker strings such that task level `docker:` just referred to appropriate docker variable.

Reply:
I put some docker variables in globally in the wdl, so check out the new convention. Thanks for this idea!

Reply:
this looks so much cleaner, nice work!